src/components/NewCommentForm.tsx

```diff
@@ -57,15 +61,14 @@
       return;
     }
 
-    setSubmitting(true);
+    dispatch(newCommentActions.setNewComment(true));
+    // setSubmitting(true);
```

Comment:
The `setSubmitting(true)` line is commented out. If you want to indicate loading during form submission, consider uncommenting this line.